crates/sparrow-scheduler/Cargo.toml

```diff
@@ -17,6 +17,7 @@ index_vec.workspace = true
 itertools.workspace = true
 serde.workspace = true
 sparrow-arrow = { path = "../sparrow-arrow" }
+sparrow-batch = { path = "../sparrow-batch" }
```

Comment:
Just a note for next time -- this probably would have been cleaner if we had a PR to split out the batch stuff and then a separate PR for the sources/interfaces (eg., separate refactoring from introducing new code). Would have made a really fast review for the refactor, and then a smaller review for the new code.